### PR TITLE
Version Selecting with color, issue #2106

### PR DESCRIPTION
**Pull request by kevin88tu**
Updated code from Hackathon

### CODE REVIEW

**Comment by kevin88tu** [comment]
Screen Shot 2020-03-12 at 8 02 51 PM
Demonstration of this code. When you try to select a version, you have to select between "v3", "v2(Multiple Choice)", and "v2(Checkbox)". Is this the desired behavior or should it only select between "v2" and "v3"?

File: pages/instructorQuestions/instructorQuestions.ejs [hunk 1]
```diff
@@ -49,6 +49,7 @@
                             <% if (locals.course_instance) { %>
                             <th data-placeholder="Choose assessment">Assessments</th>
                             <% } %>
+                            <th class="filter-select filter-exact" data-placeholder="Choose Version">Version</th>
```
Comment (mwest1066) [inline pages/instructorQuestions/instructorQuestions.ejs]:
```suggestion
                            <th class="filter-select filter-exact" data-placeholder="Choose version">Version</th>
```

**Comment by mwest1066** [comment]
> Demonstration of this code. When you try to select a version, you have to select between "v3", "v2(Multiple Choice)", and "v2(Checkbox)". Is this the desired behavior or should it only select between "v2" and "v3"?

This looks great!

**Comment by mwest1066** [comment]
Can we re-order the columns so "Version" comes before "Assessments"?

**Pull request merged**